manifest/com.__Choco__/DoublePrecision/info.json

```diff
@@ -0,0 +1,24 @@
+{
+	"name": "DoublePrecision",
+	"id": "com.__Choco__.DoublePrecision",
+	"description": "Fixes floating point rendering errors!",
+	"category": "Visual Tweaks",
+	"sourceLocation": "https://github.com/AwesomeTornado/Resonite-DoublePrecision-RML",
+	"versions": {
+		"1.6.1": {
+			"releaseUrl": "https://github.com/AwesomeTornado/Resonite-DoublePrecision-RML/releases/tag/V1.6.1",
+			"artifacts": [
+				{
+					"url": "https://github.com/AwesomeTornado/Resonite-DoublePrecision-RML/releases/download/V1.6.1/DoublePrecision.dll",

```

Comment:
In `Shaders.cs`, it seems a bit odd to be defining 27 constants for what could be done with 8 as those intermediate steps aren't being used outside of this file. Additionally could be reduced further to just the 4 for your replacement shader uri's as all of the official ones can be referenced from `OfficialAssets.Shaders.PBSTriplanarSpecular` and similar.